### PR TITLE
Changes to Capacity Metrics in OCS Persistent Storage Dashboard

### PR DESCRIPTION
**Pull request by bipuladh**
- Add Raw Capacity Card in Dashboard
 - Remove Avaiable Metric from Breakdown Card
 - Rename Capacity Breakdown Card
 - Add Info Tip to Capacity Breakdown Card
Screenshot: 
Screenshot from 2020-12-03 16-19-19
Screenshot from 2020-12-03 14-14-10
Screenshot from 2020-12-03 14-20-13

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/constants/queries.ts [hunk 1]
```diff
@@ -70,6 +73,11 @@ export const CAPACITY_USAGE_QUERIES = {
     'sum((kubelet_volume_stats_used_bytes * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info * on (storageclass)  group_left(provisioner) kube_storageclass_info {provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"}))',
 };
 
+export const CAPACITY_INFO_QUERIES = {
+  [StorageDashboardQuery.RAW_CAPACITY_TOTAL]: 'ceph_cluster_total_bytes',
+  [StorageDashboardQuery.RAW_CAPACITY_USED]: 'ceph_cluster_total_used_raw_bytes',
+};
+
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/constants/queries.ts:80]: @anmolsachan PTAL

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-breakdown/capacity-breakdown-card.tsx [hunk 2]
```diff
@@ -75,7 +74,14 @@ const BreakdownCard: React.FC<DashboardItemProps> = ({
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>{t('ceph-storage-plugin~Capacity breakdown')}</DashboardCardTitle>
+        <DashboardCardTitle>
+          {t('ceph-storage-plugin~Capacity breakdown')}
+          <FieldLevelHelp>
+            This card shows the used capacity for Usable storage, broken-down by different
+            kubernetes resources. Usable storage is all the data that can be stored in the system
+            after decreasing the replication policies.
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-breakdown/capacity-breakdown-card.tsx:82]:
```suggestion
            t(This card shows the used capacity for Usable storage, broken-down by different
            kubernetes resources. Usable storage is all the data that can be stored in the system
            after decreasing the replication policies.)
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      x: 'Available',
+      y: availableCapacity.value,
+      string: availableCapacityAdjusted.string,
+    },
+  ];
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Raw Capacity</DashboardCardTitle>
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:51]:
```suggestion
        <DashboardCardTitle>t(Raw Capacity)</DashboardCardTitle>
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+              <ChartLegend
+                fill={colorScale[1]}
+                title="Available"
+                text={availableCapacityAdjusted.string}
+              />
+            </div>
+            <div className="ceph-raw-usage__item ceph-raw-usage__chart">
+              <ChartDonut
+                ariaDesc="Available versus Used Capacity"
+                ariaTitle="Available versus Used Capacity"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:72]:
```suggestion
                ariaDesc=t('Available versus Used Capacity')
                ariaTitle=t('Available versus Used Capacity')
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+                height={150}
+                width={150}
+                data={donutData}
+                labels={({ datum }) => `${datum.string}`}
+                title={usedCapacityAdjusted.string}
+                subTitle={`Used of ${totalCapacity.string}`}
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:78]:
```suggestion
                subTitle={t('Used of {{capacity}}', {capacity: totalCapacity.string})}
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+      <div className="ceph-raw-usage-loading-legend__item skeleton-activity" />
+      <div className="ceph-raw-usage-loading-legend__item skeleton-activity" />
+    </div>
+    <div className="ceph-raw-usage-loading__chart skeleton-activity" />
+  </div>
+);
+
+const ErrorCardBody: React.FC = () => (
+  <>
+    <div className="ceph-raw-usage--error text-muted">Not Available</div>
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:108]:
```suggestion
    <div className="ceph-raw-usage--error text-muted">t('Not Available')</div>
```

**Review by afreen23 — CHANGES_REQUESTED** [review]
Externalize the strings in following format:
`ceph-storage-plugin~{string}`

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+      <DashboardCardHeader>
+        <DashboardCardTitle>{t('ceph-storage-plugin~Raw Capacity')}</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody className="ceph-raw-usage__container">
+        {!loading && !loadError && (
+          <>
+            <div className="ceph-raw-usage__item ceph-raw-usage__legend">
+              <ChartLegend
+                fill={colorScale[0]}
+                title="Used"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:61]: This as well.
```suggestion
                title="Used"
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+                text={usedCapacityAdjusted.string}
+                titleClassName="ceph-raw-card-legend__title--pad"
+              />
+              <ChartLegend
+                fill={colorScale[1]}
+                title="Available"
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx:67]: externalize this too
```suggestion
                title="Available"
```

**Review by afreen23 — APPROVED** [review]
Just two comments, approved from my end

**Pull request merged**